content/blog/2016/2016-08-31-scaling-jenkins-at-jenkins-world.adoc

```diff
@@ -0,0 +1,183 @@
+---
+layout: post
+title: "Scaling Jenkins at Jenkins World 2016"
+tags:
+- event
+- JenkinsWorld
+author: rtyler
+---
+
+[NOTE]
+--
+This is a guest post by link:https;//github.com/rtyler[R. Tyler Croy], who is a
+long-time contributor to Jenkins and the primary contact for Jenkins project
+infrastructure. He is also a Jenkins Evangelist at
+link:http://cloudbees.com[CloudBees, Inc.]
+--
+
+image:/images/conferences/Jenkins-World_125x125.png[Jenkins World, role=right]
+
+
+I find the topic of "scaling Jenkins" to be incredibly interesting because,
+more often than not, scaling Jenkins isn't just about scaling a single instance
+but rather scaling an _organization_ and its continuous delivery processes. In
+many cases when people talk about "scaling Jenkins" they're talking about
+"Jenkins as a Service" or "Continuous Delivery as a Service" which introduces a
+much broader scope, and also more organization-specific requirements, to the
+problem.
+
+One of my favorite parts of a big conference like
+link:https://www.cloudbees.com/jenkinsworld/home[Jenkins World] is getting to
+see how other people are solving similar problems at different organizations,
+in essence:
+"link:https://english.stackexchange.com/questions/120739/a-peek-into-the-sausage-factory[how
```

Comment:
Link goes to question where someone asks for alternative expressions, why? I'd have expected a definition.